share/jupyterhub/static/less/page.less

```diff
@@ -70,3 +70,7 @@
   border-color: @jupyter-orange;
   outline-color: @jupyter-orange;
 }
+
+.text-muted {
```

Comment:
Is this class required? I suspect that assigning the variable value [does this already](https://github.com/twbs/bootstrap/blob/v3.4.1/less/type.less#L105).

Reply:
I commented it out and got the same result so I have removed it.